### ISSUE TITLE
Feature Request: Add support for Token Action HUD

### ISSUE THREAD

**Issue by GrumpyVader** [trev33b/dice-tooltip2#4]
It would be a nice QoL feature to support displaying the tooltips in Token Action HUD as well. Great work by the way!

**Comment by trev33b** [comment]
That's an interesting idea.  I'm an avid user of the Token Action HUD as well.  

I probably won't do this immediately because I'll need to dig into it to see how best to work with it.  I'll put it on my mid-term list.

**Comment by trev33b** [comment]
I haven't started looking at this yet, but I have been in the midst of a complete rewrite of the code.  Once that is complete (relatively soon), I'll start seeing how viable it'll be to integrate Dice Tooltip with Token Action HUD. I do agree it would be nice to have.
